packages/web_mouse_click_boundary/pubspec.yaml

```diff
@@ -0,0 +1,54 @@
+name: web_mouse_click_boundary
+description: A new Flutter package project.
+version: 0.0.1
+author:
+homepage:
```

Comment:
Make version `0.8.0` or similar, and update the `homepage`/`repository` fields so we can publish this to pub.dev.